example/main.go

```diff
@@ -12,10 +12,10 @@ package main
 // @Security ApiKey read write
 // @SecurityScheme ApiKey apiKey header Authorization
 // @PackageAlias "example" ""
-// @Tags "Foo"
-// @Tags "Bar" "Baz"
-// @Tags "Vfoo"
-// @Tags "Foo With Spaces"
+// @Tags "Group A" "Foo"
```

Comment:
I'm not sure I like this syntax. We could also have a separate tag groups section like..
```
// @TagGroups "Group A" "Foo" "Bar"
// @TagGroups "Group B" "Vfoo" "foo With Spaces"
```

Reply:
I prefer the approach that you currently have. As the number of tags continues to grow, it makes it each tag/group more readable since each line is shorter. I imagine that the `@TagGroups` lines could get long for large groups.

Personally I'm willing to accept the duplicated group names in this format to keep the annotations more scannable.

Reply:
Yeah, another thing I liked was that it would be hard(er?) to leave a tag without a group by doing it this way.